data/postprocess/boundary-velocity-residual/box_3d_boundary_strain_rate-3d.txt

```diff
@@ -0,0 +1,22 @@
+# Test data for ascii data boundary velocity.
```

Comment:
Where do you use this file? It looks like your postprocessor expects a strain rate tensor of dimension dim-1 (i.e. strain rate on the surface), not a volume strain rate.

Reply:
Currently, I am not using it. But, the visualization postprocessor outputs boundary_strain_rate_residual of dim dimension. 
In the current implementation, the boundary_strain_rate_residual[3] is actually equivalent to strain_rate[4], if the input strain rate is 0. 
Therefore, I also created the data file with all the components and was not sure if I needed the volume strain rate in the data or not. I will remove the file in the next commit.